internal/core/CMakeLists.txt

```diff
@@ -16,15 +16,23 @@
 
 cmake_minimum_required( VERSION 3.18 )
 
+if ( APPLE )
```

Comment:
set variables within core_build.sh and internal/core/build.sh

Reply:
APPLE is a cmake variable: https://cmake.org/cmake/help/v3.0/variable/APPLE.html